VultisigApp/VultisigApp/Views/Vault/VaultDetailView.swift

```diff
@@ -41,6 +41,8 @@ struct VaultDetailView: View {
         }
         .onChange(of: vault.coins) {
             viewModel.fetchCoins(for: vault)
+            // Without the set data it does not update the balance
+            setData()
```

Comment:
I removed that ,  because if I add 10 coins for ETH , this method will be called 10 times , that means balance will updated 10 times

Reply:
Removed